incubating/jira-issue-comment/codefresh.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+type Annotation struct {
+	EntityId   string `json:"entityId"`
+	EntityType string `json:"entityType"`
+	Key        string `json:"key"`
+	Value      string `json:"value"`
+}
+
+func createBuildAnnotation(environment Config) {
+	annotation := Annotation{
+		EntityId:   environment.CodefreshBuildId,
+		EntityType: "build",
+		Key:        "jira-issue-url",
+		Value:      environment.JiraBaseUrl + "/browse/" + environment.JiraIssueId,
```

Comment:
The example of the Base URL has a trailing `/` this concat here is resulting in `https://codefresh-io.atlassian.net//browse/SA-75` I would suggest always removing the trailing `/` from JiraBaseUrl and possibly updating the examples to exclude the `/` from end of base url but handling either.

Reply:
Clarified the examples and added a trimsuffix to remove at least one / from being passed in